### PR TITLE
Update GitHub desktop fork instructions

### LINKED ISSUE #13165: GitHub desktop cannot automatically prompt me to create a fork
According to [document](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop)
When you try to use GitHub Desktop to clone a repository that you do not have write access to, GitHub Desktop will prompt you to create a fork automatically.
but my GitHub Desktop cannot automatically prompt me to create a fork.
I don't know where the setting is wrong

### PR DESCRIPTION
**Pull request by Nikkitory**
### Why: The current fork instructions for Github desktop were out of date with the latest Github desktop.

Closes: https://github.com/desktop/desktop/issues/13165 (I realize this is an issue from a different repository, however in the comment thread of that issue you will see people mentioning that the issue arises because the docs here need updating.)

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@Nikkitory Thank you so much for opening this PR! ✨ I have filed an internal issue to fix this on our end, as this portion of the documentation will need to be updated by our team 💛

Thanks again for raising a flag on this ✨

**Comment by nguyenalex836** [comment]
@Nikkitory Apologies for the confusion, but it looks like these changes may not need to be made internally 💛 I'm reopening this PR, and one of our team members will review shortly. Thank you!

File: assets/images/help/desktop/clone-choose-button-mac.png
Comment (steveward) [inline assets/images/help/desktop/clone-choose-button-mac.png:1]: Let's keep the original screenshot and add this as a new screenshot.

File: data/reusables/desktop/cloning-location-tab.md
Comment (steveward) [inline data/reusables/desktop/cloning-location-tab.md:1]: Let's keep this reusable and create a new one with the URL tab option.

File: data/reusables/desktop/cloning-repository-list.md
Comment (steveward) [inline data/reusables/desktop/cloning-repository-list.md:1]: Same as above --  let's keep this reusable as is and create a new one specific to cloning via a URL.

**Review by steveward — CHANGES_REQUESTED** [review]
Thanks for submitting this pull request @nikkitory! I've left comments for some changes I'd like to see made. Mainly, I'd like to keep the original screenshots and reusables and instead add new ones that represent the updates you were making to the originals. If you have any questions or need additional clarity, please let me know!

**Comment by warp-core** [comment]
Would it be ok if I copied the head branch from here, made the requested changes, and created a new pull request?

**Comment by nguyenalex836** [comment]
@warp-core Please do! 💛

**Pull request closed without merging**